entity-service-attribute-translator/src/main/java/org/hypertrace/entity/attribute/translator/AttributeMetadata.java

```diff
@@ -0,0 +1,16 @@
+package org.hypertrace.entity.attribute.translator;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
```

Comment:
Probably just @Value, since we don't want it mutable. Also both Value and Data include EqualsAndHashCode, can be removed.